be/src/storage/aggregate_type.h

```diff
@@ -29,7 +29,9 @@ enum StorageAggregateType {
     STORAGE_AGGREGATE_BITMAP_UNION = 7,
     // Replace if and only if added value is not null
     STORAGE_AGGREGATE_REPLACE_IF_NOT_NULL = 8,
-    STORAGE_AGGREGATE_PERCENTILE_UNION = 9
+    STORAGE_AGGREGATE_PERCENTILE_UNION = 9,
+    // used in insert ignore mode to keep fist value.
+    STORAGE_AGGREGATE_FIRST = 10
```

Comment:
What does `first` mean? The last version value? Or the first random row with the same pk in a INSERT batch?

Reply:
like agg function `first_value`